src/test/java/org/prebid/server/auction/AmpRequestFactoryTest.java

```diff
@@ -58,15 +61,15 @@
 
     @Before
     public void setUp() {
+        given(httpRequest.getParam("tag_id")).willReturn("tagId");
```

Comment:
Throughout the codebase we prefer to use `eq(<string>)` because Mockito swears if matchers and raw values are used in mock setup and verification.